pkg/cmd/root.go

```diff
@@ -26,6 +34,6 @@ var RootFlags = []cli.Flag{
 	},
 	&cli.StringFlag{
 		Name:  "endpoint",
-		Usage: "set the daemon endpoint URI (overrides .env.toml)",
+		Usage: "set the daemon endpoint `URI` (overrides .env.toml)",
```

Comment:
The backticks have syntactic significance, by the way.  https://github.com/urfave/cli/blob/master/docs/v2/manual.md#placeholder-values

Reply:
Yup, I'm precisely adding them for that reason!